src/controllers/tutor.js

```diff
@@ -1,8 +1,49 @@
+const { studentModel } = require('../models/student');
+
 module.exports = {
-  getAllTutors: async () => {},
-  getTutorDetails: async () => {},
-  filterTutorsByTags: async () => {},
-  searchForTutor: async () => {},
+  getAllTutors: async (req, res) => {
+    try {
+      const tutors = await studentModel.find({ isTutor: true });
+      res.json(tutors);
```

Comment:
Consider specifying the status code for better understanding; even though its 200 by default.

Reply:
ok ill do that
also do you think I should write the tests right away?

Reply:
I'm not sure actually